packages/common/src/grpc-service.js

```diff
@@ -227,8 +229,17 @@ GrpcService.prototype.request = function(protoOpts, reqOpts, callback) {
   delete reqOpts.autoPaginateVal;
 
   var service = this.getService_(protoOpts);
-  var grpcOpts = {};
 
+  var metadata = null;
```

Comment:
Yes, this is what I meant. Either this or the previous iteration should work fine, but what you have now is closer to how I originally expected the API to be used.